main/src/ca/uwaterloo/flix/language/errors/ResolutionError.scala

```diff
@@ -534,6 +534,37 @@ object ResolutionError {
     override def explain(formatter: Formatter): Option[String] = None
   }
 
+  /**
+    * An error indicating the number of term patterns does not match the enum tag term count.
```

Comment:
-> An error indicating that the number of arguments to a tag-pattern is mismatched.